rumbler/rumbler-portfolio
# fix: improve version-bump script

## Description
Improvements to the version bump script to fix identified issues.

### Changes
- Remove color codes from changelog
- Add new version at the beginning of changelog (without using temporary file)
- Remove automatic push to allow double-check
- Fix changelog format to match current style

### How to test
1. Run the script with `./scripts/version-bump.sh patch`
2. Check if the changelog is properly formatted
3. Confirm there are no color codes in the file
4. Verify that push is not done automatically
